### PR TITLE
Feat/add wagmi demo

### PR DESCRIPTION
**Pull request by q20274982**
## Summary

- add new codesandbox example `with-evm-wagmi`

## Related Links

**Asana**: https://app.asana.com/0/1201812548509877/1205217254151301/f

## Checklist

- [x] Pasted Asana link.
- [x] Tagged labels.

## Prerequisite/Related Pull Requests

Prerequisite #233

### CODE REVIEW

**Review by sanyu1225 — APPROVED** [review]

File: examples/with-evm-wagmi/src/components/SignTypedData.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from 'react'
+import { recoverTypedDataAddress } from 'viem'
+import { type Address, useSignTypedData } from 'wagmi'
+
+const domain = {
+  name: 'Ether Mail',
+  version: '1',
+  chainId: 1,
```
[the remark this answers is not in the document]
Comment (q20274982) [reply examples/with-evm-wagmi/src/components/SignTypedData.tsx]: resolved [feat: remove redundant component](https://github.com/blocto/blocto-sdk/pull/228/commits/0f27c228f37533f79d81ea79309d5951ac029437)

File: examples/with-evm-wagmi/src/components/SignMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    variables,
+    error,
+    isLoading,
+    signMessage,
+  } = useSignMessage()
+
+  useEffect(() => {
+    ;(async () => {
+      if (variables?.message && signature) {
+        const recoveredAddress = await recoverMessageAddress({
```
Comment (sanyu1225) [inline examples/with-evm-wagmi/src/components/SignMessage.tsx:18]: this function throw error

File: examples/with-evm-wagmi/src/components/SignTypedData.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    message,
+    primaryType: 'Mail',
+  })
+
+  const [recoveredAddress, setRecoveredAddress] = useState<Address>()
+  useEffect(() => {
+    if (!data) return
+    ;(async () => {
+      setRecoveredAddress(
+        await recoverTypedDataAddress({
```
Comment (sanyu1225) [inline examples/with-evm-wagmi/src/components/SignTypedData.tsx:48]: this function throw error

File: examples/with-evm-wagmi/src/components/SignMessage.tsx (hunk 2 above)
Comment (q20274982) [reply examples/with-evm-wagmi/src/components/SignMessage.tsx]: fixed in [fix: remove recoverTypedDataAddress & recoverMessageAddress](https://github.com/blocto/blocto-sdk/pull/228/commits/3f15d41250be27fb9c834872d99b691015280eb2)

**Review by sanyu1225 — APPROVED** [review]
LGTM

**Pull request merged**